hw2/converter.js

```diff
@@ -0,0 +1,20 @@
+globalThis.converter=function(arr){
```

Comment:
Math complexity of this algorithm = O(5n). 
When Array Length will be 100 - we iterate 500 times

Reply:
Fixed.
If I'm not mistaken, now math complexity of this algorithm = O(n) ...